src/Tribe/RSVP/Early_Access/Update_Notice.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+namespace Tribe\Tickets\RSVP\Early_Access;
+
+use Tribe__Tickets__Main;
+use Tribe__Settings;
+use Tribe__Admin__Notices;
+
+class Update_Notice {
+
+	/**
+	 * @var Early_Access
+	 */
+	private $early_access;
+
+	/**
+	 * The minimum version of Event Tickets needed to
+	 * display the update notice.
+	 *`
+	 * @since TBD
+	 */
```

Comment:
:no_entry_sign: There must be exactly one blank line before the tags in a doc comment

<sub>hash: e22016b1b417ddf7140e6f4cceb19e12</sub>